circle.yml

```diff
@@ -6,6 +6,7 @@ machine:
     PARALLEL_TEST_PROCESSORS: 8
 
     PATH: $HOME/.cabal/bin/:$PATH
+    GO_PROJECT: "$HOME/.go_workspace/src/github.com/Originate/gt"
```

Comment:
Yeah, that was another question I had for you. The repo is called `git-town`, but I want the tool to be called `gt`. I haven't found a way to tell `go install` to name the binary `gt` instead of `git-town`. 

Am I doing this wrong? I think configuring the binary name should be supported. What if a code base wants to create 2 binaries, let's say separate `compress` and `decompress` binaries?